### ISSUE TITLE
Wither storm not showing up on my friends screen when playing with it.

### ISSUE THREAD

**Issue by Carterminator11** [nonamecrackers2/crackers-wither-storm-mod#1808]
### Minecraft Version

1.20.1

### Mod Version

4.0

### Mod Loader

Forge

### Mod Loader Version

47.1.44

### Describe the bug

Every time we try to summon the wither storm. My friend who also has the mod couldn't see it when being spawned, moving or summoned. But somehow I see it on my screen. I really don't know how to fix this and me and my friend tried everything.

### Is OptiFine installed?

- [ ] OptiFine is installed

### Mod List

Farmers delight
Guard Villagers
Butchers Delight
Geckolib 4
Weeping Angels
cave_dweller
From the Fog
The Graveyard
Small Ships
The twilight forest
The Man From the Fog
Recruits mod
Essential

**Comment by FireCode125** [comment]
Optifine is installed, there’s your problem.

**Comment by BluePineappleGuy** [comment]
> Optifine is installed, there’s your problem.

he didn't mention it in the report

**Comment by nonamecrackers2** [comment]
> Optifine is installed, there’s your problem.

Yeah, also, just because it’s installed doesn’t automatically mean it’s the issue. Yes, optifine causes some compatibility issues, but not all the time.

For OP, is that your friends mod list? If not, can you send it?

**Comment by watersofnazareth** [comment]
Closing due to lack of information
